### ISSUE TITLE
wslServer.sh stays running after VS Code is shut down

### ISSUE THREAD

**Issue by bradwilson** [microsoft/vscode#183086]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.78.2 (user setup)
- OS Version: Windows_NT x64 10.0.22621

Steps to Reproduce:

1. Install VS Code with the `ms-vscode-remote.remote-wsl` extension
1. Open a WSL 2 distribution. Nothing unexpected running.

   ![image](https://github.com/microsoft/vscode/assets/16944/14e80b04-0393-4299-a25f-cfe2bef9df9a)

1. Open a folder with `code .` from a WSL 2 terminal window (can even be an empty folder). See there are three new PTYs created; `wslServer.sh` is the root process in pts/2 (23975), and node is running as the root process in pts/3 (24052) and pts/4 (24168):

   ![image](https://github.com/microsoft/vscode/assets/16944/a75fcc0c-757a-43e3-be5d-4a2edc060a32)

1. Close VS Code. pts/3 and /4 are terminated. `wslServer.sh` from pts/2 is still alive:

   ![image](https://github.com/microsoft/vscode/assets/16944/e5124a82-551a-4b10-a5ad-7f69d8fdf2ae)

   This process will stay running seemingly forever (several hours is the most patience I had for it). It'll keep your WSL 2 distribution alive as well.

1. Opening another VS Code instance actually launches _another_ `wslServer.sh` is another new PTY:

   ![image](https://github.com/microsoft/vscode/assets/16944/77cd03bd-4e93-4adf-a9db-31166ae78d06)

Basically, every time I launch and quit VS Code, I have to go clean up after it and kill the `wslServer.sh` process.

This feels like a fairly recent issue (last month or three, maybe; I don't use WSL regular enough to give a better time frame unfortunately).

**Comment by aeschli** [comment]
Thanks for the issue, fix is on the way... (upcoming version 0.78.7)
